osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -98,15 +100,24 @@ private void updateMods()
             }
 
             MultiplierLabel.Text = $"{multiplier:N2}x";
-            if (!ranked)
-                MultiplierLabel.Text += " (Unranked)";
-
             if (multiplier > 1.0)
                 MultiplierLabel.FadeColour(HighMultiplierColour, 200);
             else if (multiplier < 1.0)
                 MultiplierLabel.FadeColour(LowMultiplierColour, 200);
             else
                 MultiplierLabel.FadeColour(Color4.White, 200);
+
+            RankedLabel.Text = null;
+            if (!ranked)
+            {
+                RankedLabel.Text += " (Unranked)";

```

Comment:
`=` should be enough here to set. Also the leading whitespace could be removed and the label could have an appropriate distance instead.

Reply:
You said a conflicting review. If I do `RankedLabel.Text = "(Unranked)"` with no space, it overrides `RankedLabel.Text = " "` on the top. It's best that I replace only the `+=` with `=`.

Reply:
There is no `RankedLabel.Text = " "` though? D: Am I blind?

Reply:
> Also the leading whitespace could be removed and the label could have an appropriate distance instead.

Maybe I'm confused. What is this asking exactly?

Reply:
Have `RankedLabel.Text = "(Unranked)";` and eventually move the label to the right with padding if it doesn't look appropriate.

Reply:
*margin